src/accelerate/utils/other.py

```diff
@@ -59,7 +59,9 @@ def is_compiled_module(module):
     return isinstance(module, torch._dynamo.eval_frame.OptimizedModule)
 
 
-def extract_model_from_parallel(model, keep_fp32_wrapper: bool = True, recursive: bool = False):
+def extract_model_from_parallel(
+    model, keep_fp32_wrapper: bool = True, keep_torch_compile: bool = True, recursive: bool = False
+):
```

Comment:
I think that you want to keep keep_torch_compile=False for a few version before setting it to True @muellerzr ?

Reply:
I thought setting `keep_torch_compile=True` is keeping it as-is since the current version keeps compilation.

Reply:
Oh yeah sorry I misread that, you can ignore my comment.